### DISCUSSION TITLE
Collabora Office not working after update

### DISCUSSION THREAD

**Discussion by Sulmuza** [nextcloud/all-in-one#3658] [Questions] [answered]
So I am not sure what went wrong but I think Collabora is trying to use IPV6 instead of IPV4 

Here is the output of 

sudo docker exec -it nextcloud-aio-nextcloud bash
curl -vvv https://$NC_DOMAIN:443/hosting/discovery
exit

e3e142107e80:/var/www/html# curl -vvv https://$NC_DOMAIN:443/hosting/discovery
* processing: https://redacted/hosting/discovery
*   Trying 192.168.178.xxx:443...
* connect to 192.168.178.xxx port 443 failed: Connection refused
*   Trying [2606:4700:3034::6815:28a8]:443...
* Immediate connect fail for 2606:4700:3034::6815:28a8: Address not available
*   Trying [2606:4700:3031::ac43:bb73]:443...
* Immediate connect fail for 2606:4700:3031::ac43:bb73: Address not available
* Failed to connect to nc.com port 443 after 200 ms: Couldn't connect to server
* Closing connection

**Comment by szaimen** [comment, marked as answer]
> * processing: [redacted/hosting/discovery](https://redacted/hosting/discovery)
> * Trying 192.168.178.xxx:443...
> * connect to 192.168.178.xxx port 443 failed: Connection refused
> * Trying [2606:4700:3034::6815:28a8]:443...
> * Immediate connect fail for 2606:4700:3034::6815:28a8: Address not available
> * Trying [2606:4700:3031::ac43:bb73]:443...
> * Immediate connect fail for 2606:4700:3031::ac43:bb73: Address not available
> * Failed to connect to nc.com port 443 after 200 ms: Couldn't connect to server
> * Closing connection

Seems like you found the problem that you need to fix in your networking infrastructure.

**Reply by Sulmuza** [reply]
what should I fix? everything except collabora works fine.

**Reply by szaimen** [reply]
Apparently dns is not configured correctly in your network

**Reply by Sulmuza** [reply]
thank you very much it was indeed my dns. I has a false dns entry for my website in my local dns server
